packages/app-admin-ui/client/components/Nav/index.js

```diff
@@ -237,19 +237,22 @@ function PrimaryNavItems({
   let onRenderIndexPage = () => {
     hasRenderedIndexPage = true;
   };
+
   let pageNavItems =
     pages && pages.length
-      ? pages.map(node =>
-          renderChildren(
-            node,
-            authListKey,
-            mouseIsOverNav,
-            getListByKey,
-            adminPath,
-            0,
-            onRenderIndexPage
+      ? pages
+          .filter(node => node.addToNav !== false)
+          .map(node =>
+            renderChildren(
+              node,
+              authListKey,
+              mouseIsOverNav,
+              getListByKey,
+              adminPath,
+              0,
+              onRenderIndexPage
+            )
```

Comment:
Let's leave these changes to happen in #1906

Reply:
That's just going to create a merge conflict for one of us. It's a one liner can we leave it here?